pkg/docker/docker.go

```diff
@@ -69,8 +66,8 @@ const (
 	// timeout is reached, certain Docker API calls might error out.
 	DefaultDockerTimeout = 60 * time.Second
 
-	// DefaultShmSize is the default shared memory size to use (in bytes) if not specified.
-	DefaultShmSize = int64(1024 * 1024 * 64)
+	// defaultShmSize is the default shared memory size to use (in bytes) if not specified.
+	defaultShmSize = int64(1024 * 1024 * 64)
```

Comment:
ah ... I now get you decided to mark it private.  I would leave it public.  No harm with it being accessible, and I would not be shocked if something else needs this in the future.  Also I wonder if there is origin specific code that is looking it up.